app/app.test.js

```diff
@@ -17,18 +18,28 @@ describe('GET',() => {
             .get('/api/topics')
             .expect(200)
             .then((response) => {
-                console.log(response)
+                // console.log(response)
                response.body.topics.forEach((topic) => {
                 //testing for a truthy value to see if these properties exist
                 expect(typeof topic.description).toBe("string")
                 expect(typeof topic.slug).toBe("string")
                })
             })
         });
+  })
+})
+
+describe('GET',() => {
+    describe('/api', () => {
+        test('return with an object describing all endpoints available ', async () => {
+            const response = await request(app).get('/api')
+            expect(200)
+            expect(response.body).toEqual(endpoints)
+        });
   
-        test('Test Case 2', () => {
- 
-     });
+        //  struggling on how to implement error handling  
```

Comment:
Don't worry, there's no need for error testing on this because there isn't anything a user can do to make this go wrong 👍